spec/components/candidate_interface/links_to_previous_applications_component_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::LinksToPreviousApplicationsComponent do
+  let(:candidate) { build(:candidate) }
+  let(:application_form) { create(:application_form, candidate: candidate, phase: :apply_2) }
+  let(:previous_application_form) { create(:completed_application_form, candidate: candidate, phase: :apply_2, submitted_at: 3.days.ago) }
+  let(:first_application_form) { create(:completed_application_form, candidate: candidate, phase: :apply_1, submitted_at: 5.days.ago) }
+
+  before do
+    application_form
+    previous_application_form
+    first_application_form
+  end
```

Comment:
FYI `let!` does what you want here:

```suggestion
  let!(:application_form) { create(:application_form, candidate: candidate, phase: :apply_2) }
  let!(:previous_application_form) { create(:completed_application_form, candidate: candidate, phase: :apply_2, submitted_at: 3.days.ago) }
  let!(:first_application_form) { create(:completed_application_form, candidate: candidate, phase: :apply_1, submitted_at: 5.days.ago) }
```

Reply:
Okay cool. On Find banging lets was a capital crime haha. Good to know i can use them here.